src/tree/CosmosDBAccountWizard/CosmosDBAccountCreateStep.ts

```diff
@@ -48,8 +50,7 @@ export class CosmosDBAccountCreateStep extends AzureWizardExecuteStep<ICosmosDBW
             options.capabilities?.push({ name: SERVERLESS_CAPABILITY_NAME });
         }
 
-        const response: DatabaseAccountsCreateOrUpdateResponse = await client.databaseAccounts.createOrUpdate(rgName, accountName, options);
-        context.databaseAccount = response._response.parsedBody;
+        context.databaseAccount = await client.databaseAccounts.beginCreateOrUpdateAndWait(rgName, accountName, options);
         ext.outputChannel.appendLog(`Successfully created Cosmos DB account "${accountName}".`);
```

Comment:
`_response._parsedBody` was used a couple of times and seems like that no longer exists and we can just use the response from the SDK now.